Source/WebCore/dom/Node.h

```diff
@@ -141,11 +141,11 @@ class Node : public EventTarget, public CanMakeCheckedPtr {
     inline RefPtr<ContainerNode> protectedParentNode() const; // Defined in ContainerNode.h.
     static ptrdiff_t parentNodeMemoryOffset() { return OBJECT_OFFSETOF(Node, m_parentNode); }
     inline Element* parentElement() const;
-    Node* previousSibling() const { return m_previous.get(); }
-    RefPtr<Node> protectedPreviousSibling() const { return m_previous.get(); }
+    Node* previousSibling() const { return m_previous; }
+    RefPtr<Node> protectedPreviousSibling() const { return m_previous; }
```

Comment:
I wonder if it would be useful to name `previousSibling()` something scary like `rawPreviousSibling()` or `unsafePreviousSibling()` or something. (Ditto for `nextSibling()`).

Reply:
I don't currently have this pattern in WebKit. `foo()` returns a raw pointer and `protectedFoo()` returns a RefPtr / Ref. This is our current pattern.